### ISSUE TITLE
Document private data policy on issues

### ISSUE THREAD

**Issue by flbulgarelli** [mumuki/mumuki-platform#191]

**Comment by flbulgarelli** [comment]
TIP I have just learned: if you add a `CONTRIBUTING.md` file to the root of the project  - I guess a plain `CONTRIBUTING` may also work - ,  when you create an issue,  you are prompted to read guidelines first.

**Comment by faloi** [comment]
Great. I'll add a CONTRIBUTING file both in English and Spanish.

**Comment by flbulgarelli** [comment]
Here is a piece of an old contribution guidelines, which I have just extracted from the old wiki: 

```
So, you want to fix an issue or add a feature? 

Here are a few things you should consider:

1. Check our backlog at https://waffle.io/mumuki/mumuki-platform, Ready cards are those we consider most important.
2. Start a new branch from `master`, following one of these naming conventions:
  - prepend the issue number, like: `210-no-submissions`
  - include a hash (#) and the issue number in any part of the name, like: `feature-#210-no-submissions`
3. Push the new branch, so everybody knows that you're working on that.
4. Start a pull request, including the [special keyword syntax](https://help.github.com/articles/closing-issues-via-commit-messages/) (eg. `fixes #210` or `closes #210`) in its body. Please give us a little explanation about what you've done, including a screenshot of the UI if it changed.
```

Perhaps it could be used as a base version of it.
